### DISCUSSION TITLE
Why kaniko is not an officially supported Google product

### DISCUSSION THREAD

**Discussion by kennhhhhh** [GoogleContainerTools/kaniko#3184] [Q&A]
why **kaniko** is not officially supported?

**Comment by abitrolly** [comment]
Some ideas.

1. Google don't have money to hire people to support this
2. Google doesn't use it
3. People @ Google don't want `kaniko` to be killed by PM folks
4. There is a reputation risk to accept contributions from dogs on internet
5. The code is so awful that security team said it will quit if they have to support this
6. Mercurial in retrograde
7. Maintainers want to have some fun after they retire
